libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/tasks/Tasks.kt

```diff
@@ -200,7 +200,7 @@ abstract class AbstractKotlinCompile<T : CommonCompilerArguments> : AbstractKotl
                 }
             ).disallowChanges()
             task.taskBuildCacheableOutputDirectory.value(getKotlinBuildDir(task).map { it.dir("cacheable") }).disallowChanges()
-            task.taskBuildLocalStateDirectory.value(getKotlinBuildDir(task).map { it.dir("localstate") }).disallowChanges()
+            task.taskBuildLocalStateDirectory.value(getKotlinBuildDir(task).map { it.dir("local-state") }).disallowChanges()
```

Comment:
Curious - what is the point of such directory renaming? :thinking:

Reply:
Good catch! :) There was an IDE inspection warning with "localstate": Typo: In word 'localstate'

Also, another directory next to it is called "classpath-snapshot", so I wanted to have a consistent naming.